std/examples/curl.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+async function curl(url: string): Promise<void> {
+  const res = await fetch(url);
+  await Deno.copy(Deno.stdout, res.body);
+}
```

Comment:
Make this return a `Reader` and export it. The copy can happen below.

Reply:
Yeah I cribbed this from `std/examples/cat.ts`.

So no changes required?

Reply:
No, you can resolve this thread.

Reply:
Thanks